iot/api-client/manager/src/test/java/com/example/cloud/iot/examples/ManagerIT.java

```diff
@@ -413,32 +413,50 @@ public void testHttpDeviceConfig() throws Exception {
   // MQTT device tests
   @Test
   public void testMqttDeviceConfig() throws Exception {
```

Comment:
Please add the command test as a separate test and leave the configuration test as-is.